### DISCUSSION TITLE
Why was deleted the option "Access Type" ?

### DISCUSSION THREAD

**Discussion by jmzavala** [keycloak/keycloak#14470] [Archive]
I upgraded from 18.0 to 19.0.1 and I don't have the "Access Type" selection anymore to config a Client. I was trying to config an only-bearer client, and it was imposible without making an import. 

Option deleted:
![Screen Shot 2022-09-16 at 21 56 20](https://user-images.githubusercontent.com/1740571/190835126-a23a57e8-10ab-4f96-946f-fa1b7cb9eee6.png)

I didn't find any entries in the release notes related with this.

Why was  it deleted? 
Are there any security or architectures reasons for that?

Thanks!!

**Comment by sschu** [comment]
You can just disable all flows and you have a bearer-only client. This redundancy was the reason for removal AFAIK.
